hyperbolic-rs/rudders/models/euclidean.py

```diff
@@ -37,14 +37,18 @@ class SMFactor(BaseEuclidean):
     """Simple Matrix Factorization model."""
 
     def score(self, user_embeds, item_embeds, all_pairs):
-        """Score based on dot product"""
+        """Score based on dot product (cosine similarity)"""
```

Comment:
Thanks for pointing out the "cosine similarity" part!
For hyperbolic embeddings, the norm plays a crucial role, so a baseline that does not remove the norm aspect from these Euclidean embeddings might be interesting to look at as well?

Reply:
I'll try this out in the new set of models/experiments